src/models/article.model.js

```diff
@@ -23,6 +23,7 @@ const articleSchema = new mongoose.Schema(
     violence_type: {
       type: [String],
       required: true,
+      validate: (v) => v == null || (v.length > 0),
```

Comment:
may be specify what is v here?

Reply:
what do u mean exactly? writing validator instead of v ?

Reply:
i just did not understand what is "v"? is it "validator"? i thought it is "value" or something

Reply:
but never mind, it is just a small comment cuz it is not clear. But lets change in this PR mock names (like @thdvbr commented) for your article test and merge, so we can see what's left.
Do not forget to check lint (CI is not working for this command now)

Reply:
changed